.github/workflows/molecule.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
           python-version: '3.x'
 
       - name: Install test dependencies.
-        run: pip3 install ansible ansible-lint yamllint docker molecule-docker "molecule[docker,lint]"
+        run: pip3 install ansible ansible-compat==0.5.0 ansible-lint yamllint docker molecule-docker "molecule[docker,lint]"
```

Comment:
Downgrade ansible-compat as a later release introduced a breaking change whereby the dependent roles needed for the Molecule tests aren't installed into the local cache and hence can't be located. This can be reverted once the fix in ansible-compat has been released.